### PR TITLE
Color all deletion options consistenly red

### LINKED ISSUE #6892: Remove Selected button on Entities panel not red
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

On the Entities panel when multi-selecting enemies, the Remove Selected button should be red because it is performing a destructive action.

image

## Expected behavior

Red button

## Steps to reproduce

1. Go to Configuration -> Entities
2. Click the checkbox next to one or more Entities
3. Look at button action row in top right above the data table... Remove should be red

## Environment

- Home Assistant release with the issue: 0.115.0b3
- Last working Home Assistant release (if known): ??
- Browser and browser version: Chrome stable
- Operating system: macOS

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

In the existing dialogs the delete option is always colored red. To create consistency this PR does:
1. Colors the deletion icon for automations, scenes, scripts red
2. Adjusts the color for the automation action row from yellow to red (was red for almost all instances already before but one)
3. Color the "Remove Selected" entity config button (fixes #6892) 

image

image

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #6892
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/script/ha-script-editor.ts [hunk 1]
```diff
@@ -446,6 +447,9 @@ export class HaScriptEditor extends LitElement {
         mwc-fab.dirty {
           bottom: 0;
         }
+        .warning {
```
Comment (ludeeus) [inline src/panels/config/script/ha-script-editor.ts:450]: No need to include it here, haStyle already has it

File: src/panels/config/scene/ha-scene-editor.ts [hunk 2]
```diff
@@ -781,6 +782,9 @@ export class HaSceneEditor extends SubscribeMixin(LitElement) {
         mwc-fab.dirty {
           bottom: 0;
         }
+        .warning {
```
Comment (ludeeus) [inline src/panels/config/scene/ha-scene-editor.ts:785]: No need to include it here, haStyle already has it

File: src/panels/config/automation/ha-automation-editor.ts [hunk 3]
```diff
@@ -578,7 +579,11 @@ export class HaAutomationEditor extends LitElement {
         }
         mwc-fab.dirty {
           bottom: 0;
+        }        
+        .warning {
```
Comment (ludeeus) [inline src/panels/config/automation/ha-automation-editor.ts:583]: No need to include it here, haStyle already has it

File: src/panels/config/automation/action/ha-automation-action-row.ts [hunk 4]
```diff
@@ -336,7 +336,7 @@ export default class HaAutomationActionRow extends LitElement {
           --mdc-theme-text-primary-on-background: var(--disabled-text-color);
         }
         .warning {
-          color: var(--warning-color);
+          color: var(--error-color);
```
Comment (ludeeus) [inline src/panels/config/automation/action/ha-automation-action-row.ts:339]:
```suggestion
```

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: src/panels/config/script/ha-script-editor.ts (hunk 1 above)
Comment (spacegaier) [reply src/panels/config/script/ha-script-editor.ts:450]: Thanks for the hint! Commit coming in a few secs.

**Comment by zsarnett** [comment]
https://github.com/home-assistant/frontend/issues/6892

Can you get this as well

**Comment by spacegaier** [comment]
> 
> 
> #6892
> 
> Can you get this as well

@zsarnett Done.

File: src/panels/config/automation/ha-automation-editor.ts [hunk 5]
```diff
@@ -579,6 +580,7 @@ export class HaAutomationEditor extends LitElement {
         mwc-fab.dirty {
           bottom: 0;
         }
+      }
```
Comment (ludeeus) [inline src/panels/config/automation/ha-automation-editor.ts:583]: leftover bracket?

Comment (spacegaier) [reply src/panels/config/automation/ha-automation-editor.ts]: Indeed 😑 . Fixed.

**Review by zsarnett — APPROVED** [review]

**Review by ludeeus — APPROVED** [review]

**Pull request merged**